### DISCUSSION TITLE
Export replay files

### DISCUSSION THREAD

**Discussion by Speykious** [ppy/osu#13565] [Ideas] [answered]
Is it currently possible in osu!lazer to export replay files, so that we can share them with friends or say turn them into a video using an external solution? If not, how difficult would it be to implement that feature?

I don't know the internal structure of a replay file (although would be interested in an explanation just out of curiosity), but if I had to predict any problems, I would assume that there must at least be an incompatibility issue for mods that currently aren't on Stable... Although I really have no clue.

**Comment by Joehuu** [comment, marked as answer]
You can already export replays by right clicking a score in song select, stable's f2 export when in results screen is not implemented yet.

![osu_2021-06-18_13-01-39](https://user-images.githubusercontent.com/35318437/122611073-8b122b80-d035-11eb-8d52-761f83c6de4d.jpg)

**Reply by Speykious** [reply]
:0 Thank you! It worked!
